buildpipeline/pipeline.json

```diff
@@ -20,7 +20,7 @@
           "Parameters": {
             "PB_DockerTag": "centos-7-d485f41-20173404063424",
             "PB_AdditionalBuildArguments":"-PortableBuild=true -strip-symbols -SkipTests=$(PB_SkipTests)",
-            "PB_AdditionalMSBuildArguments":"/p:DotNetRestoreSources=$(PB_RestoreSource)",
+            "PB_AdditionalMSBuildArguments":"/p:DotNetRestoreSources=$(PB_RestoreSource) /p:ProduceStablePackages=$(PB_IsStable) /p:PackagePreReleaseLabel=$(PB_VersionStamp)",
```

Comment:
Wait, is `ProduceStablePackages` our own parameter in this repo?  Why not just set `StabilizePackageVersion` and `PreReleaseLabel` properties here?

Reply:
=1